e2e/test/scenarios/sharing/sharing-reproductions.cy.spec.js

```diff
@@ -630,15 +630,15 @@ describeEE("issue 26988", () => {
     });
 
     openStaticEmbeddingModal({
-      activeTab: "appearance",
+      activeTab: "lookAndFeel",
       previewMode: "preview",
       acceptTerms: false,
     });
 
     cy.wait("@previewDashboard");
     getIframeBody().should("have.css", "font-family", "Lato, sans-serif");
 
-    cy.findByLabelText("Playing with appearance options")
+    cy.findByLabelText("Customizing look and feel")
```

Comment:
Since the removal of the second section, all the settings belong to the first section now.